src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs

```diff
@@ -2,170 +2,104 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
+
 namespace System.Drawing.Drawing2D
 {
-    using System.Runtime.InteropServices;
-
-    /// <include file='doc\AdjustableArrowCap.uex' path='docs/doc[@for="AdjustableArrowCap"]/*' />
-    /// <devdoc>
-    ///    Represents an adjustable arrow-shaped line
-    ///    cap.
-    /// </devdoc>
     public sealed class AdjustableArrowCap : CustomLineCap
     {
-        internal AdjustableArrowCap(IntPtr nativeCap) :
-            base(nativeCap)
-        { }
-
-        /// <include file='doc\AdjustableArrowCap.uex' path='docs/doc[@for="AdjustableArrowCap.AdjustableArrowCap"]/*' />
-        /// <devdoc>
-        ///    Initializes a new instance of the <see cref='System.Drawing.Drawing2D.AdjustableArrowCap'/> class with the specified width and
-        ///    height.
-        /// </devdoc>
-        public AdjustableArrowCap(float width,
-                                  float height) :
-            this(width, height, true)
-        { }
-
-        /// <include file='doc\AdjustableArrowCap.uex' path='docs/doc[@for="AdjustableArrowCap.AdjustableArrowCap1"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Drawing.Drawing2D.AdjustableArrowCap'/> class with the specified width,
-        ///       height, and fill property.
-        ///    </para>
-        /// </devdoc>
-        public AdjustableArrowCap(float width,
-                                  float height,
-                                  bool isFilled)
-        {
-            IntPtr nativeCap = IntPtr.Zero;
-
-            int status = SafeNativeMethods.Gdip.GdipCreateAdjustableArrowCap(
-                                height, width, isFilled, out nativeCap);
-
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
+        internal AdjustableArrowCap(IntPtr nativeCap) : base(nativeCap) { }
 
-            SetNativeLineCap(nativeCap);
-        }
-
-        private void _SetHeight(float height)
-        {
-            int status = SafeNativeMethods.Gdip.GdipSetAdjustableArrowCapHeight(new HandleRef(this, nativeCap), height);
-
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
-        }
+        public AdjustableArrowCap(float width, float height) : this(width, height, true) { }
 
-        private float _GetHeight()
+        public AdjustableArrowCap(float width, float height, bool isFilled)
         {
-            float height;
-            int status = SafeNativeMethods.Gdip.GdipGetAdjustableArrowCapHeight(new HandleRef(this, nativeCap), out height);
+            int status = SafeNativeMethods.Gdip.GdipCreateAdjustableArrowCap(height, width, isFilled, out IntPtr nativeCap);
```

Comment:
Oh yeah sorry I should have reverted these changes - they were sent in before your comments elsewhere